README.md

```diff
@@ -6,13 +6,15 @@
 
 * * *
 
-<img width="64" src="https://raw.githubusercontent.com/laurent22/joplin/master/Assets/LinuxIcons/256x256.png" align="left" /> **Joplin** is a free, open source note taking and to-do application, which can handle a large number of notes organised into notebooks. The notes are searchable, can be copied, tagged and modified either from the applications directly or from your own text editor. The notes are in [Markdown format](#markdown).
+<img width="64" src="https://raw.githubusercontent.com/laurent22/joplin/master/Assets/LinuxIcons/256x256.png" align="left" /> **Joplin** is a free, open source note taking and to-do list application available for Windows, Linux, macOS, Android, and iOS. It can easily handle a vast number of notes, which are organised into notebooks. Notes can be searched, copied, tagged, and edited -- even with your favourite text [editor](#External-text-editor).
```

Comment:
I reworked the intro text for readability, logical flow, etc. and added a link to the section about editing externally.